components/director/internal/domain/label/repository.go

```diff
@@ -170,6 +170,30 @@ func (r *repository) DeleteAll(ctx context.Context, tenant string, objectType mo
 	return errors.Wrapf(err, "while deleting all Label entities from database for %s %s", objectType, objectID)
 }
 
+func (r *repository) DeleteByKey(ctx context.Context, tenant string, key string) error {
```

Comment:
Please rebase to latest master to not have changes from other PR.

Reply:
done